gobblin-temporal/src/main/java/org/apache/gobblin/temporal/workflows/helloworld/GreetingWorkflowImpl.java

```diff
@@ -41,16 +50,20 @@ public class GreetingWorkflowImpl implements GreetingWorkflow {
      *
      * The activity options that were defined above are passed in as a parameter.
      */
-    private final FormatActivity activity = Workflow.newActivityStub(FormatActivity.class, options);
+    private final FormatActivity formatActivity = Workflow.newActivityStub(FormatActivity.class, options);
+    private final SubmitGTEActivity
+        timerActivity = Workflow.newActivityStub(SubmitGTEActivity.class, options);
```

Comment:
aside from `options`, the `SubmitGTEActivityImpl` seems purely plumbing.  given that, does it really make sense to require every caller to construct their own?

perhaps merely have the `TemporalEventTimer.Factory` constructor take in `ActivityOptions`, so the details of `SubmitGTEActivity` remain encapsulated.  given the dynamic resolution by `Workflow.newActivityStub(SubmitGTEActivity.class, opts)`, I'm not sure there would ever even be the need to use a custom invocation.  if so, that's an argument in favor of such encapsulation.